### PR TITLE
Migrated OTP api to Verify api

### PR DESCRIPTION
**Pull request by dysosmus**
@samwierema breaking changes since the OTP Api doesn't exist anymore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,11 +128,11 @@ client.otp_generate(31612345678, {:reference => "YourReference"})
...
 ```
 
-This sends a token to the recipient, which can be verified with the **otp_verify** method.
+This sends a token to the recipient, which can be verified with the **verify_token** method.
 
 ```ruby
-# otp_verify requires a recipient, a token as required parameters, and other optional paramaters
-client.otp_verify(31612345678, 123456, {:reference => "YourReference"})
+# verify_token requires the id of the verify request and a token as required parameters.
+client.verify_token(31612345678, 123456)
```
Comment (nicolas-brousse) [inline README.md]: First argument must be a verify id instead the phone number I guess

**Comment by cristiangreco** [comment]
Hey 👋 thanks for working on this PR. Do you guys have any plan to have it merged soon?

**Comment by p0wl** [comment]
thanks! can you also publish a new version (on github and rubygems) =)

**Comment by marcelcorso** [comment]
@p0wl 👍  
releasing is on @samwierema's hands.

**Comment by p0wl** [comment]
is there a way to test the verify API with a `test_` api token? like a fixed otp that always works?

**Comment by marcelcorso** [comment]
@p0wl I don't think so but shoot an email to our friends at support@messagebird.com and they can tell you for sure.

**Pull request merged**